torch/testing/_internal/composite_compliance.py

```diff
@@ -24,7 +24,7 @@ def check_attr_consistency(wrapper_tensor, metadata_name, metadata_accessor):
     if metadata_wrapper_tensor == metadata_elem:
         return
     raise RuntimeError(
-        f"This operator is not CompositeImplicitAutograd compliant: the "
+        f"This operator is not Composite Compliant: the "
```

Comment:
Name change because "CompositeImplicitAutograd" only applies to operators and many backward formulas are just regular c++ functions that consist of pytorch operations